.gitmodules

```diff
@@ -13,3 +13,6 @@
 [submodule "packages/contracts/lib/forge-std"]
 	path = packages/contracts/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "packages/contracts/lib/solidity-stringutils"]
```

Comment:
> we don't want solidity-stringutils library installed for diamond implementation. Imo, we should be able to implement without that.

solidity-stringutils is used for unit test, not on diamond contract.

Reply:
we should be able to do everything without that even in unit test

Reply:
![image](https://user-images.githubusercontent.com/5579669/201190986-83bc27fc-43bf-4fc4-9729-404c8115360b.png)
To test adding new facet feature on forge test, we need to get function selectors of that facet.
At this time, we should manipulate the string that is generated when performing foreign function call (ffi) in solidity.
And Solidity-stringutils makes it super easy

Reply:
We should be able to print them using abi and keccak like abi.encodeWithSelector if this is the main reason.

Reply:
> We should be able to print them using abi and keccak like abi.encodeWithSelector if this is the main reason.

Yeah, I understand your intention not to introduce more libraries
But the purpose was not to print them, just to manipulate the dumping data printed from foreign function call in order to get function selectors as well as manage them in solidity code

Please check the implementation of stringutils in contracts/test/helpers/DiamondTestHelper.sol

If I tell again, 
To test adding new facet feature, need to get function selectors of that facet, and to manipulate the selectors. It's a bit easy in ethersJS, but not in solidity code. Afaik there is no better way to get function selectors of any facet and to manipulate them in solidity code rather than current implementation

Reply:
As I mentioned in several comments, we should keep this conversation `unresolved` until you remove both external libraries from `.gitmodules` and `lib` folder. In terms of strict test for function selectors, I already let you know how to do.